pyart/graph/radardisplay.py

```diff
@@ -112,6 +116,7 @@ def __init__(self, radar, shift=(0.0, 0.0)):
         units = radar.time['units']
         calendar = radar.time['calendar']
         self.time_begin = netCDF4.num2date(times, units, calendar)
+        self.times = datetimes_from_radar(radar)
```

Comment:
I think the calculation of times should be delayed until needed, not added as a RadarDisplay attribute.